sw/device/lib/testing/test_framework/ottf_isrs.h

```diff
@@ -29,7 +29,7 @@ typedef enum ottf_exc_id {
  */
 typedef enum ottf_internal_irq_id {
   kInternalIrqLoadInteg = 0xffffffe0,
-  kInternalIrqNmi = 0x0000001f
+  kInternalIrqNmi = 0x8000001f
```

Comment:
@timothytrippel my reading of the spec is that bit31 always has to be set for interrupts (including nmi), let me know what you think. The hardware behaves this way as well, but that could be "two wrongs make a right"

Reply:
yep that looks to be typo, you are correct, at least thats what is stated [here](https://ibex-core.readthedocs.io/en/latest/03_reference/exception_interrupts.html) in the Ibex docs, and in the RV spec [here](https://people.eecs.berkeley.edu/~krste/papers/riscv-privileged-v1.9.pdf) on page 34; section 3.1.20.

Nice catch, sorry about that.